rocAL/rocAL/source/meta_data/meta_data_graph_factory.cpp

```diff
@@ -38,6 +38,10 @@ std::shared_ptr<MetaDataGraph> create_meta_data_graph(const MetaDataConfig& conf
         {
             return std::make_shared<BoundingBoxGraph>();
         }
+        case MetaDataType::PolygonMask:
```

Comment:
can you please combine 3 cases into single switch branch since they are returning the same data